notanorm/model.py

```diff
@@ -39,6 +52,7 @@ class DbCol(NamedTuple):
     notnull: bool = False  # not null
     fixed: bool = False  # not varchar
     default: Any = None  # has a default value
+    custom: DbColCustomInfo = None  # dialect-specific column information
```

Comment:
Wondering if maybe this should take a tuple -- e.g. if you're writing a schema by hand and want to have multiple vendor-specific things.

Probably don't need right now though, since it's easy to add in the future.

Reply:
i see, yes, that could be necessary.   i agree, easy enough to add later